spec/models/mixins/authentication_mixin_spec.rb

```diff
@@ -102,17 +102,19 @@ def self.name; "TestClass"; end
     end
 
     it "should be triggered for kubernetes" do
-      auth = AuthToken.new(:name => "bearer", :auth_key => "valid-token")
-      FactoryBot.create(:ems_kubernetes, :authentications => [auth])
+      auths = FactoryBot.create_list(:auth_token, 1, :name => "bearer")
+      auths.first.auth_key = "valid_token" # Assign this afterwards to force an update callback
+      FactoryBot.create(:ems_kubernetes, :authentications => auths)
```

Comment:
This spec change indicates a problematic change in behavior to me. I don't think this should be changed in this PR.

Just creating the authentication (without a subsequent update) should trigger an authentication check.